WebARonARKit/WebARonARKit.js

```diff
@@ -1019,11 +1049,21 @@
       WebARonARKitVRDisplay.projectionMatrix_[i] = data.projectionMatrix[i];
     }
 
-    // Only on iOS!
-    if (!window.ARCore) {
-      callRafCallbacks();
-      nativeAdvanceFrame();
+    // Did the native side pass the camera frame? Then update the image!
+    if (window.WebARonARKitUsesCameraFrames && data.cameraFrame !== "") {
+      WebARonARKitVRDisplay.passThroughCameraImage_.src = data.cameraFrame;
+      // The raf callbacks and the advance of the frame will be done once the
+      // image is loaded (see the image load event above)
+    }
+    else {
+      // Only on iOS and when there is no camera frame, call the raf callbacks
+      // and advance a frame
+      if (!window.ARCore) {
```

Comment:
why would this code even exist on an ARCore device? it gets injected by the experimental browser code

Reply:
I am trying to reuse the same polyfill for both ARKit and ARCore webview based prototypes.

Reply:
ah right, cool!